draft-ietf-quic-multipath.md

```diff
@@ -842,15 +822,10 @@ Path ID 1), and server provides two connection IDs
 
 Before the client opens a new path by sending a packet on that path
 with a PATH_CHALLENGE frame, it has to check whether there is
-an unused connection IDs available for each side.
-In this example, the client chooses the connection ID S2
-as the Destination Connection ID in the new path.
+an unused connection IDs for the same unused Path ID available for each side.
+Respectively, the client chooses the connection ID S1
+as the Destination Connection ID of the new path.
 
-If the client has used all the allocated connection IDs, it is supposed to retire
-those that are not used anymore, and the server is supposed to provide
-replacements, as specified in {{QUIC-TRANSPORT}}.
-Usually, it is desired to provide one more connection ID as currently
-in use, to allow for new paths or migration.
```

Comment:
This was moved into section 4.3.3 ([Allocating, Consuming and Retiring Connection IDs](https://quicwg.org/multipath/draft-ietf-quic-multipath.html#name-allocating-consuming-and-re))